ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -57,24 +55,24 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <form.input
-    @name='issuer'
-    @type='text'
+  <Hds::Form::TextInput::Field
```

Comment:
can we add `isoptional` attr here?

Reply:
@DhariniJeeva `@isOptional` added back in last commit.  Thank you for the catch